geojson_pydantic/geometries.py

```diff
@@ -207,6 +207,14 @@ def _wkt_coordinates(self) -> str:
             f"({_lines_wtk_coordinates(polygon)})" for polygon in self.coordinates
         )
 
+    @validator("coordinates")
```

Comment:
In creating a test that should have failed I realized this wasn't actually constrained before and it should have been.